### DISCUSSION TITLE
MAPL 2.20.0

### DISCUSSION THREAD

**Discussion by mathomp4** [GEOS-ESM/MAPL#1487] [Announcements]
This is a zero-diff minor release of MAPL. Two new features have been added for couplers and ExtData2G as well as various fixes.

---

### Fixed

- Removed one redundant get_file_extension call
- Fix issue where ACG was called when no file had changed
- Add missing `rc=status` in `MAPL_GetResourceFromMAPL_scalar`
- Fixed bugs with next generation ExtData
- Fixed variable PTR40 declaration in GenericCplComp.F90

### Added

- Added support for 4d variables in the coupler. Intentionally decided not to support 4d in the coupler's ReadRestart and WriteRestart to catch errors
- Added ability to use multiple rules for different time periods in next generation ExtData

### Changed

- Cleaned up a bit of old CMake
- Updated CircleCI config to use new orb `build` job
- Updated `components.yaml` to match GEOSgcm v10.22.1
  - ESMA_env v3.13.0
  - ESMA_cmake v3.12.0

---

## What's Changed
* Fix issue with calling ACG multiple times by @mathomp4 in https://github.com/GEOS-ESM/MAPL/pull/1448
* Update CI to use orb build job, Update components.yaml by @mathomp4 in https://github.com/GEOS-ESM/MAPL/pull/1450
* GitFlow: Handmerge main into develop  by @mathomp4 in https://github.com/GEOS-ESM/MAPL/pull/1458
* GitFlow: Handmerge main into develop 2022-Mar-299 by @mathomp4 in https://github.com/GEOS-ESM/MAPL/pull/1463
* Add missing rc=status by @mathomp4 in https://github.com/GEOS-ESM/MAPL/pull/1467
* Fixes #1455. Added support for 4d variables except in coupler's readR… by @atrayano in https://github.com/GEOS-ESM/MAPL/pull/1469
* Fix typo in mapl_tree.py by @mathomp4 in https://github.com/GEOS-ESM/MAPL/pull/1471
* ExtData2G updates, bugs fixes and multiple filestream feature. by @bena-nasa in https://github.com/GEOS-ESM/MAPL/pull/1475
* remove one redundant get_file_extension call by @weiyuan-jiang in https://github.com/GEOS-ESM/MAPL/pull/1480
* Fixes #1483. Fixed the rank of variable declaration by @atrayano in https://github.com/GEOS-ESM/MAPL/pull/1484
* Prepare for 2.20.0 Release by @mathomp4 in https://github.com/GEOS-ESM/MAPL/pull/1481
* more infomation if get_var fails by @weiyuan-jiang in https://github.com/GEOS-ESM/MAPL/pull/1339
* Merge develop into main for 2.20.0 release by @mathomp4 in https://github.com/GEOS-ESM/MAPL/pull/1486

**Full Changelog**: https://github.com/GEOS-ESM/MAPL/compare/v2.19.2...v2.20.0

<hr /><em>This discussion was created from the release <a href='https://github.com/GEOS-ESM/MAPL/releases/tag/v2.20.0'>MAPL 2.20.0</a>.</em>

**Comment by atrayano** [comment]
Mike,

I hope the fix I suggested yesterday for MAPL is working for you.

Also wanted to give you an update that we have released MAPL v2.20.0 which contains the fix needed for you plus other small fixes

Atanas

From: Matthew Thompson ***@***.***>
Reply-To: GEOS-ESM/MAPL ***@***.***>
Date: Tuesday, April 19, 2022 at 11:54 AM
To: GEOS-ESM/MAPL ***@***.***>
Cc: "Trayanov, Atanas L. (GSFC-610.1)[SCIENCE SYSTEMS AND APPLICATIONS INC]" ***@***.***>, Mention ***@***.***>
Subject: [EXTERNAL] [GEOS-ESM/MAPL] MAPL 2.20.0 (Discussion #1487)


This is a zero-diff minor release of MAPL. Two new features have been added for couplers and ExtData2G as well as various fixes.

________________________________
From CHANGELOG.md
Fixed

  *   Removed one redundant get_file_extension call
  *   Fix issue where ACG was called when no file had changed
  *   Add missing rc=status in MAPL_GetResourceFromMAPL_scalar
  *   Fixed bugs with next generation ExtData
  *   Fixed variable PTR40 declaration in GenericCplComp.F90

Added

  *   Added support for 4d variables in the coupler. Intentionally decided not to support 4d in the coupler's ReadRestart and WriteRestart to catch errors
  *   Added ability to use multiple rules for different time periods in next generation ExtData

Changed

  *   Cleaned up a bit of old CMake
  *   Updated CircleCI config to use new orb build job
  *   Updated components.yaml to match GEOSgcm v10.22.1
     *   ESMA_env v3.13.0
     *   ESMA_cmake v3.12.0

________________________________
What's Changed

  *   Fix issue with calling ACG multiple times by @mathomp4<https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fmathomp4&data=05%7C01%7CAtanas.L.Trayanov%40nasa.gov%7Ca2088acdadc3447782cc08da221ceee6%7C7005d45845be48ae8140d43da96dd17b%7C0%7C0%7C637859804909444697%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=oW8FlbzJIb2mX0hk6aXgiMqp2VK2lxJ385wGMC0WDG8%3D&reserved=0> in #1448<https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FGEOS-ESM%2FMAPL%2Fpull%2F1448&data=05%7C01%7CAtanas.L.Trayanov%40nasa.gov%7Ca2088acdadc3447782cc08da221ceee6%7C7005d45845be48ae8140d43da96dd17b%7C0%7C0%7C637859804909444697%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=QUtiMilo95DuG%2FVBRss0X3VOGRazwT9Emlb92TcNdFs%3D&reserved=0>
  *   Update CI to use orb build job, Update components.yaml by @mathomp4<https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fmathomp4&data=05%7C01%7CAtanas.L.Trayanov%40nasa.gov%7Ca2088acdadc3447782cc08da221ceee6%7C7005d45845be48ae8140d43da96dd17b%7C0%7C0%7C637859804909444697%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=oW8FlbzJIb2mX0hk6aXgiMqp2VK2lxJ385wGMC0WDG8%3D&reserved=0> in #1450<https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FGEOS-ESM%2FMAPL%2Fpull%2F1450&data=05%7C01%7CAtanas.L.Trayanov%40nasa.gov%7Ca2088acdadc3447782cc08da221ceee6%7C7005d45845be48ae8140d43da96dd17b%7C0%7C0%7C637859804909444697%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=UuYB42VmuVzFef3TbE4xmXBPjh3vEdyD4aLhi%2BrjMCs%3D&reserved=0>
  *   GitFlow: Handmerge main into develop by @mathomp4<https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fmathomp4&data=05%7C01%7CAtanas.L.Trayanov%40nasa.gov%7Ca2088acdadc3447782cc08da221ceee6%7C7005d45845be48ae8140d43da96dd17b%7C0%7C0%7C637859804909444697%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=oW8FlbzJIb2mX0hk6aXgiMqp2VK2lxJ385wGMC0WDG8%3D&reserved=0> in #1458<https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FGEOS-ESM%2FMAPL%2Fpull%2F1458&data=05%7C01%7CAtanas.L.Trayanov%40nasa.gov%7Ca2088acdadc3447782cc08da221ceee6%7C7005d45845be48ae8140d43da96dd17b%7C0%7C0%7C637859804909444697%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=Xbl5eId4vgTceHchT1UGeVvjdGYy08RzqYYbS8gdWEE%3D&reserved=0>
  *   GitFlow: Handmerge main into develop 2022-Mar-299 by @mathomp4<https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fmathomp4&data=05%7C01%7CAtanas.L.Trayanov%40nasa.gov%7Ca2088acdadc3447782cc08da221ceee6%7C7005d45845be48ae8140d43da96dd17b%7C0%7C0%7C637859804909444697%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=oW8FlbzJIb2mX0hk6aXgiMqp2VK2lxJ385wGMC0WDG8%3D&reserved=0> in #1463<https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FGEOS-ESM%2FMAPL%2Fpull%2F1463&data=05%7C01%7CAtanas.L.Trayanov%40nasa.gov%7Ca2088acdadc3447782cc08da221ceee6%7C7005d45845be48ae8140d43da96dd17b%7C0%7C0%7C637859804909444697%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=NkOBF2KZ8GHe%2BTT2kqqmYyu9Esqqn6KpNwcfqwoTf3k%3D&reserved=0>
  *   Add missing rc=status by @mathomp4<https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fmathomp4&data=05%7C01%7CAtanas.L.Trayanov%40nasa.gov%7Ca2088acdadc3447782cc08da221ceee6%7C7005d45845be48ae8140d43da96dd17b%7C0%7C0%7C637859804909444697%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=oW8FlbzJIb2mX0hk6aXgiMqp2VK2lxJ385wGMC0WDG8%3D&reserved=0> in #1467<https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FGEOS-ESM%2FMAPL%2Fpull%2F1467&data=05%7C01%7CAtanas.L.Trayanov%40nasa.gov%7Ca2088acdadc3447782cc08da221ceee6%7C7005d45845be48ae8140d43da96dd17b%7C0%7C0%7C637859804909444697%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=tNVUbkTX%2FjyYajGVaYfg%2B6E4%2F3KwG3RLLHuUi8sf5Yk%3D&reserved=0>
  *   Fixes #1455<https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FGEOS-ESM%2FMAPL%2Fissues%2F1455&data=05%7C01%7CAtanas.L.Trayanov%40nasa.gov%7Ca2088acdadc3447782cc08da221ceee6%7C7005d45845be48ae8140d43da96dd17b%7C0%7C0%7C637859804909444697%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=XWEvFhx06ecx76yGYM2buf6J%2ByPhtVL02qGQcAhnaTo%3D&reserved=0>. Added support for 4d variables except in coupler's readR… by @atrayano<https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fatrayano&data=05%7C01%7CAtanas.L.Trayanov%40nasa.gov%7Ca2088acdadc3447782cc08da221ceee6%7C7005d45845be48ae8140d43da96dd17b%7C0%7C0%7C637859804909444697%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=sX0M85beu6dVaOUaH6L%2Frg3DXB85H5Ovftp5dBdelfI%3D&reserved=0> in #1469<https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FGEOS-ESM%2FMAPL%2Fpull%2F1469&data=05%7C01%7CAtanas.L.Trayanov%40nasa.gov%7Ca2088acdadc3447782cc08da221ceee6%7C7005d45845be48ae8140d43da96dd17b%7C0%7C0%7C637859804909444697%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=RpTzC5xQ8KgdS0Z%2FGmqE6hh6KFeTU%2BB7asdTmPLV4Lg%3D&reserved=0>
  *   Fix typo in mapl_tree.py by @mathomp4<https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fmathomp4&data=05%7C01%7CAtanas.L.Trayanov%40nasa.gov%7Ca2088acdadc3447782cc08da221ceee6%7C7005d45845be48ae8140d43da96dd17b%7C0%7C0%7C637859804909444697%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=oW8FlbzJIb2mX0hk6aXgiMqp2VK2lxJ385wGMC0WDG8%3D&reserved=0> in #1471<https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FGEOS-ESM%2FMAPL%2Fpull%2F1471&data=05%7C01%7CAtanas.L.Trayanov%40nasa.gov%7Ca2088acdadc3447782cc08da221ceee6%7C7005d45845be48ae8140d43da96dd17b%7C0%7C0%7C637859804909444697%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=H4vFGdkrSmK26gILaZsGIW2ohUu2uAJNPQeXF%2FHNaeM%3D&reserved=0>
  *   ExtData2G updates, bugs fixes and multiple filestream feature. by @bena-nasa<https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fbena-nasa&data=05%7C01%7CAtanas.L.Trayanov%40nasa.gov%7Ca2088acdadc3447782cc08da221ceee6%7C7005d45845be48ae8140d43da96dd17b%7C0%7C0%7C637859804909444697%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=t5SXNq32mn95wo75eFtqVAmsf7Nh85NTZDWgkCJJlHU%3D&reserved=0> in #1475<https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FGEOS-ESM%2FMAPL%2Fpull%2F1475&data=05%7C01%7CAtanas.L.Trayanov%40nasa.gov%7Ca2088acdadc3447782cc08da221ceee6%7C7005d45845be48ae8140d43da96dd17b%7C0%7C0%7C637859804909444697%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=YkMROY053vdhI6ZuR7%2FdfToN4E87059HtK%2FPiLBAxyA%3D&reserved=0>
  *   remove one redundant get_file_extension call by @weiyuan-jiang<https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fweiyuan-jiang&data=05%7C01%7CAtanas.L.Trayanov%40nasa.gov%7Ca2088acdadc3447782cc08da221ceee6%7C7005d45845be48ae8140d43da96dd17b%7C0%7C0%7C637859804909444697%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=u3tdJUNMm0ZTB7tGpCDX6FNnywZnTsQ%2Bw5nIgyk25pQ%3D&reserved=0> in #1480<https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FGEOS-ESM%2FMAPL%2Fpull%2F1480&data=05%7C01%7CAtanas.L.Trayanov%40nasa.gov%7Ca2088acdadc3447782cc08da221ceee6%7C7005d45845be48ae8140d43da96dd17b%7C0%7C0%7C637859804909444697%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=IzbuBhu5qizOq63ipuoDdaW4uKo8KMzjJTVpU4y2VV0%3D&reserved=0>
  *   Fixes #1483<https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FGEOS-ESM%2FMAPL%2Fissues%2F1483&data=05%7C01%7CAtanas.L.Trayanov%40nasa.gov%7Ca2088acdadc3447782cc08da221ceee6%7C7005d45845be48ae8140d43da96dd17b%7C0%7C0%7C637859804909444697%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=SkG6vEXFjUhmRophbg%2BkmuxuoN31XVMZLbmuwZD%2BdXE%3D&reserved=0>. Fixed the rank of variable declaration by @atrayano<https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fatrayano&data=05%7C01%7CAtanas.L.Trayanov%40nasa.gov%7Ca2088acdadc3447782cc08da221ceee6%7C7005d45845be48ae8140d43da96dd17b%7C0%7C0%7C637859804909600923%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=MWXOXvLZWePewFHm6zd6%2FqwbjyrDeMFD7Li%2F7Q8EKhQ%3D&reserved=0> in #1484<https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FGEOS-ESM%2FMAPL%2Fpull%2F1484&data=05%7C01%7CAtanas.L.Trayanov%40nasa.gov%7Ca2088acdadc3447782cc08da221ceee6%7C7005d45845be48ae8140d43da96dd17b%7C0%7C0%7C637859804909600923%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=qKiZIsS0lMj9DuKmmoGxrbFq4m0Cz8VO7NCOrhPw%2Fis%3D&reserved=0>
  *   Prepare for 2.20.0 Release by @mathomp4<https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fmathomp4&data=05%7C01%7CAtanas.L.Trayanov%40nasa.gov%7Ca2088acdadc3447782cc08da221ceee6%7C7005d45845be48ae8140d43da96dd17b%7C0%7C0%7C637859804909600923%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=bV1fdb7ruTKOSFbSj1WAnjF6nenme9M4ZctyoviGYVQ%3D&reserved=0> in #1481<https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FGEOS-ESM%2FMAPL%2Fpull%2F1481&data=05%7C01%7CAtanas.L.Trayanov%40nasa.gov%7Ca2088acdadc3447782cc08da221ceee6%7C7005d45845be48ae8140d43da96dd17b%7C0%7C0%7C637859804909600923%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=%2FgC58ysHQwIecFGvBBmEwOrpwB0MEOA%2FoglFMxQszWM%3D&reserved=0>
  *   more infomation if get_var fails by @weiyuan-jiang<https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fweiyuan-jiang&data=05%7C01%7CAtanas.L.Trayanov%40nasa.gov%7Ca2088acdadc3447782cc08da221ceee6%7C7005d45845be48ae8140d43da96dd17b%7C0%7C0%7C637859804909600923%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=A3LEP0DWnsSBEkWW5%2F%2F4yBNDFhikUgSE21IzRRZ6kEg%3D&reserved=0> in #1339<https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FGEOS-ESM%2FMAPL%2Fpull%2F1339&data=05%7C01%7CAtanas.L.Trayanov%40nasa.gov%7Ca2088acdadc3447782cc08da221ceee6%7C7005d45845be48ae8140d43da96dd17b%7C0%7C0%7C637859804909600923%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=tflmv0s9q1AXOTyeN6kRAgq5Gzllla2%2BIqplbe2w0a8%3D&reserved=0>
  *   Merge develop into main for 2.20.0 release by @mathomp4<https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fmathomp4&data=05%7C01%7CAtanas.L.Trayanov%40nasa.gov%7Ca2088acdadc3447782cc08da221ceee6%7C7005d45845be48ae8140d43da96dd17b%7C0%7C0%7C637859804909600923%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=bV1fdb7ruTKOSFbSj1WAnjF6nenme9M4ZctyoviGYVQ%3D&reserved=0> in #1486<https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FGEOS-ESM%2FMAPL%2Fpull%2F1486&data=05%7C01%7CAtanas.L.Trayanov%40nasa.gov%7Ca2088acdadc3447782cc08da221ceee6%7C7005d45845be48ae8140d43da96dd17b%7C0%7C0%7C637859804909600923%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=5gI0RuDJYpWkChku7cOt%2FmJxuUIjRykLn0uS2Y2fNTU%3D&reserved=0>

Full Changelog: v2.19.2...v2.20.0<https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FGEOS-ESM%2FMAPL%2Fcompare%2Fv2.19.2...v2.20.0&data=05%7C01%7CAtanas.L.Trayanov%40nasa.gov%7Ca2088acdadc3447782cc08da221ceee6%7C7005d45845be48ae8140d43da96dd17b%7C0%7C0%7C637859804909600923%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=nfntrkJqShrohFVGiIBvFNb3U33ZaMZDP0iVEGkOvIk%3D&reserved=0>

________________________________
This discussion was created from the release MAPL 2.20.0<https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FGEOS-ESM%2FMAPL%2Freleases%2Ftag%2Fv2.20.0&data=05%7C01%7CAtanas.L.Trayanov%40nasa.gov%7Ca2088acdadc3447782cc08da221ceee6%7C7005d45845be48ae8140d43da96dd17b%7C0%7C0%7C637859804909600923%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=4dimjmVMnW09OXuvhyIUKElAJ5HsLEENU9dLk74nqf8%3D&reserved=0>.
